Jint.Benchmark/ListInteropBenchmark.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System.Collections;
+using BenchmarkDotNet.Attributes;
+using Jint.Native;
+using Jint.Runtime.Interop;
+
+namespace Jint.Benchmark;
+
+[MemoryDiagnoser]
+public class ListInteropBenchmark
+{
+    private static bool IsArrayLike(Type type)
+    {
+        if (typeof(IDictionary).IsAssignableFrom(type))
+        {
+            return false;
+        }
+
+        if (typeof(ICollection).IsAssignableFrom(type))
+        {
+            return true;
+        }
+
+        foreach (var typeInterface in type.GetInterfaces())
+        {
+            if (!typeInterface.IsGenericType)
+            {
+                continue;
+            }
+
+            if (typeInterface.GetGenericTypeDefinition() == typeof(IReadOnlyCollection<>)
+                || typeInterface.GetGenericTypeDefinition() == typeof(ICollection<>))
+            {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    private const int Count = 10_00;
```

Comment:
I used [sharplab ](https://sharplab.io/#v2:EYLgtghglgdgNAFxAJwK7wCYgNQB8ACATAAwCwAUEQIwX7EAE+VALANwW1UCcAFFcQH1ixAJSsgA)to be sure what it was ;) Now I am even more confused by the location of the `_`

Reply:
Hah, probably did some testing and forgot to remove the underscore which should be used as a thousand separator 😆